common/src/main/java/dev/architectury/networking/simple/SimpleNetworkManager.java

```diff
@@ -48,35 +51,55 @@ private SimpleNetworkManager(String namespace) {
         this.namespace = namespace;
     }
     
+    /**
+     * Convenience method to register a server -&gt; client message with no packet transformers.
+     *
+     * @see #registerS2C(String, MessageDecoder, List)
+     */
+    public MessageType registerS2C(String id, MessageDecoder<BaseS2CMessage> decoder) {
+        return registerS2C(id, decoder, List.of());
+    }
+    
     /**
      * Registers a server -&gt; client message.
      *
-     * @param id      a unique ID for the message, must be a valid value for {@link ResourceLocation#getPath}
-     * @param decoder the message decoder for the message
+     * @param id           a unique ID for the message, must be a valid value for {@link ResourceLocation#getPath}
+     * @param decoder      the message decoder for the message
+     * @param transformers a list of packet transformers to apply to the message packet
      * @return a {@link MessageType} describing the registered message
      */
-    public MessageType registerS2C(String id, MessageDecoder<BaseS2CMessage> decoder) {
+    public MessageType registerS2C(String id, MessageDecoder<BaseS2CMessage> decoder, List<PacketTransformer> transformers) {
```

Comment:
Please mark this as Experimental, I think that's also the case on the NetworkManager methods, we can reevaluate this in the future